reference/requirements.rst

```diff
@@ -29,7 +29,7 @@ Required
 .. caution::
 
     Be aware that Symfony has some known limitations when using a PHP version
-    less than 5.3.8 or equal to 5.3.16. For more information see the
+    equal to 5.3.16. For more information see the
```

Comment:
This sentence has now become a bit strange. What about: *"Be aware that Symfony has some known limitations when using PHP 5.3.16."* ?